api/src/db/migrations/v144_add_raffle_bingo_games.ts

```diff
@@ -0,0 +1,805 @@
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v143_to_v144 = (v143Data: v143UserData): v144UserData => {
+  return {
+    ...v143Data,
+    businesses: Object.fromEntries(
+      Object.values(v143Data.businesses)
+        .map((business: v143Business) => migrate_v143Business_to_v144Business(business))
+        .map((currBusiness: v144Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 144,
+  } as v144UserData;
+};
+
+export const migrate_v143Business_to_v144Business = (business: v143Business): v144Business => {
+  return {
+    ...business,
+    profileData: {
+      ...business.profileData,
+      raffleBingoGames: undefined,
+    },
+  } as v144Business;
+};
+
+export interface v144IndustrySpecificData {
+  liquorLicense: boolean;
+  requiresCpa: boolean;
+  homeBasedBusiness: boolean | undefined;
+  providesStaffingService: boolean;
+  certifiedInteriorDesigner: boolean;
+  realEstateAppraisalManagement: boolean;
+  cannabisLicenseType: v144CannabisLicenseType;
+  cannabisMicrobusiness: boolean | undefined;
+  constructionRenovationPlan: boolean | undefined;
+  carService: v144CarServiceType | undefined;
+  interstateTransport: boolean;
+  interstateLogistics: boolean | undefined;
+  interstateMoving: boolean | undefined;
+  isChildcareForSixOrMore: boolean | undefined;
+  petCareHousing: boolean | undefined;
+  willSellPetCareItems: boolean | undefined;
+  constructionType: v144ConstructionType;
+  residentialConstructionType: v144ResidentialConstructionType;
+  employmentPersonnelServiceType: v144EmploymentAndPersonnelServicesType;
+  employmentPlacementType: v144EmploymentPlacementType;
+  carnivalRideOwningBusiness: boolean | undefined;
+  propertyLeaseType: v144PropertyLeaseType;
+  hasThreeOrMoreRentalUnits: boolean | undefined;
+}
+
+export type v144PropertyLeaseType = "SHORT_TERM_RENTAL" | "LONG_TERM_RENTAL" | "BOTH" | undefined;
+
+// ---------------- v144 types ----------------
+type v144TaskProgress = "NOT_STARTED" | "IN_PROGRESS" | "COMPLETED";
+type v144OnboardingFormProgress = "UNSTARTED" | "COMPLETED";
+type v144ABExperience = "ExperienceA" | "ExperienceB";
+
+export interface v144UserData {
+  user: v144BusinessUser;
+  version: number;
+  lastUpdatedISO: string;
+  dateCreatedISO: string;
+  versionWhenCreated: number;
+  businesses: Record<string, v144Business>;
+  currentBusinessId: string;
+}
+
+export interface v144Business {
+  id: string;
+  dateCreatedISO: string;
+  lastUpdatedISO: string;
+  profileData: v144ProfileData;
+  onboardingFormProgress: v144OnboardingFormProgress;
+  taskProgress: Record<string, v144TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v144LicenseData | undefined;
+  preferences: v144Preferences;
+  taxFilingData: v144TaxFilingData;
+  formationData: v144FormationData;
+}
+
+export interface v144ProfileData extends v144IndustrySpecificData {
```

Comment:
[boulder] I just caught this, you need to include `raffleBingoGames` in the `v144ProfileData` type definition.